generators/server/templates/src/main/java/package/config/CacheConfiguration.java.ejs

```diff
@@ -491,7 +491,7 @@ public class CacheConfiguration {
                 cacheManager.getCache("oAuth2Authentication").getAdvancedCache(), this,
                 ConfigurationAdapter.create()));
             <%_ } _%>
-            <%_ if (!skipUserManagement || authenticationType === 'oauth2') { _%>
+            <%_ if (!skipUserManagement && authenticationType === 'oauth2') { _%>
```

Comment:
The condition should be `<%_ if (!skipUserManagement || (authenticationType === 'oauth2' && applicationType === 'monolith')) { _%>` to match the one from earlier in the file.

Reply:
This is because OAuth2 monoliths require this section (the failing Travis test is an OAuth2 monolith).  It compiles fine, but when you attempt to login it fails because the cache doesn't exist.

Reply:
FWIW, the OAuth logic between monoliths and microservices will soon be the same. As soon as I get https://github.com/jhipster/generator-jhipster/pull/7106 fixed.